code/game/objects/effects/decals/contraband.dm

```diff
@@ -362,6 +365,35 @@
 	desc = "The Griffin commands you to be the worst you can be. Will you?"
 	icon_state = "poster33"
 
+/obj/structure/sign/poster/contraband/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/toy/crayon/red/syndicate))
+		to_chat(usr, "<span class = 'notice'>There is no reason to change the message of this poster.</span>")
+		return
+	else
+		..()
+
+/obj/structure/sign/poster
+	var/corrupted_icon
+	var/corrupted_desc
+	var/corrupted_name
+
+/obj/structure/sign/poster/official/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/toy/crayon/red/syndicate))
+		var/obj/item/toy/crayon/red/syndicate/C = I
+		visible_message("<span class = 'notice'>You start drawing over the [src] with your crayon.</span>", "<span class = 'notice'>[user] starts drawing over the [src].</span>")
+		if(do_after(user, 50, target = src))
+			C.uses -= 5
+			icon_state = corrupted_icon
+			desc = corrupted_desc
+			name = corrupted_name
+			update_icon()
+			visible_message("<span class = 'notice'>You finish drawing over the [src] with your crayon.</span>", "<span class = 'notice'>[user] finishes drawing over the [src].</span>")
+			if(C.uses <= 0)
+				to_chat(usr, "<span class = 'notice'>The crayon is worn down to a nub.</span>")
```

Comment:
`user` here.
Spacing as well in the spans. Standard is to use something like `<span class='notice>...</span>`. Same for the others